### DISCUSSION TITLE
How to get the score of a live tennis game

### DISCUSSION THREAD

**Discussion by kam06** [cloudbet/docs#53] [General]
Hello all, 

How can I get please the score of a live tennis game ? 

Thanks

**Comment by kgravenreuth** [comment]
Hi @kam06 ,

As mentioned in [this discussion](https://github.com/Cloudbet/docs/discussions/19), we can provide you information about how your API bets have resulted via the [Bet History](https://www.cloudbet.com/api/?utm_source=github.com&utm_medium=community&utm_campaign=axis&urls.primaryName=Trading#/Trading/BetHistory) endpoint of our Trading API. 

Note that you need to be registered as a Cloudbet player to use the Trading API key with a minimum of 10 euro deposit as mentioned in our [Trading API blog article](https://www.cloudbet.com/en/blog/posts/sports-betting-api-tutorial-with-golang#toc-0?utm_source=github.com&utm_medium=community&utm_campaign=axis)

For example, If you'd like to know the result of a given event, then you can place bets on the Winner or Moneyline markets for that event and then check the bet results to know the event's results.

We are currently not planning to add results or scores to the Feed API, while we thank you for your suggestions here.

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Comment by kam06** [comment]
Hi @kgravenreuth 

Thanks for your answer. 
So basically I can know the status BetResponse (WIN, LOSS, ACCEPTED ...) but I can not get the scores of an ACCEPTED bet ? 

Thanks
Regards

**Reply by kgravenreuth** [reply]
Yes, although you can use the bet response to know event results. For example, if you'd like to know the result of a given event, then you can place bets on the Winner or Moneyline markets for that event and then check the bet results to know the event's results. We are not planning to add scores to our API at the moment.
